src/loader/AMASSLoader.py

```diff
@@ -0,0 +1,273 @@
+import glob
+import json
+import os
+import random
+from datetime import datetime
+import numpy as np
+import shutil
+
+import mathutils
+
+from src.loader.LoaderInterface import LoaderInterface
+from src.utility.Utility import Utility
+from src.utility.LabelIdMapping import LabelIdMapping
+
+import torch
+from human_body_prior.body_model.body_model import BodyModel
+
+
+class AMASSLoader(LoaderInterface):
+    """
+    AMASS is a large dataset of human motions. it unifies multiple datasets by fitting the SMPL body model to mocap (
+    motion capture) markers. The dataset includes SMPL-H body shapes and poses as well as DMPL soft tissue motions
+    parameters for every frame in every motion sequence for the sub datasets. any human pose recorded in these
+    motions could be reconstructed using the following parameters: sub dataset identifier, sequence id, frame id and
+    model gender which will represent the pose, these parameters specify the exact pose to be generated.
+
+    As for all loaders it is possible to add custom properties to the loaded object, for that use add_properties.
+
+    Finally it sets all objects to have a category_id corresponding to the void class,
+    so it wouldn't trigger an exception in the SegMapRenderer.
+
+    Note: if this module is used with another loader that loads objects with semantic mapping, make sure the other
+    module is loaded first in the config file.
+
```

Comment:
It would be great if you can show here one or two short config examples, like we did [here](https://github.com/DLR-RM/BlenderProc/blob/master/src/loader/TextureLoader.py).

Reply:
added